.github/scripts/get-component-release-notes.js

```diff
@@ -21,11 +21,11 @@ module.exports = ({ github, core }) => {
             if (issueCommentBody.includes("#Release#")) {
                 let components = issueCommentBody.split("\n")
                 const releaseIdx = components.indexOf("#Release#")
-                components = components.splice(releaseIdx + 1, components.length - 1)
-                const regex = /[A-Za-z-_0-9]+\|(https:\/\/github\.com\/.*tree.*){1}\|(https:\/\/github\.com\/.*releases.*){1}/gm;
+                components = components.splice(releaseIdx + 1, components.length)
+                const regex = /[A-Za-z-_0-9]+\|(https:\/\/github\.com\/.*tree.*){1}\|(https:\/\/github\.com\/.*releases.*){1}/;
```

Comment:
For further improvements, unrelated to the PR: you probably do not need {1} to match exactly one piece. It also makes sence to match / before and after `tree` and `releases` to avoid matching substrings, hypothetically